.github/workflows/ecosystem-ci-selected.yml

```diff
@@ -9,7 +9,21 @@ env:
 on:
   workflow_dispatch:
     inputs:
-      test:
+      ref:
+        description: 'vite ref to use'
+        required: true
+        type: string
+        default: 'main'
+      refType:
```

Comment:
Should `ref` and `refType` be inverted here? So the flow is to decide what kind of ref you use and then you give the id for it?